target/linux/ath79/dts/qca9563_nec_wg800hp.dts

```diff
@@ -0,0 +1,196 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "qca956x.dtsi"
+
+/ {
+	model = "NEC Aterm WG800HP";
+	compatible = "nec,wg800hp", "qca,qca9563";
+
+	chosen {
+		bootargs = "console=ttyS0,115200n8";
+	};
+
+	aliases {
+		led-boot = &power_green;
+		led-failsafe = &power_red;
+		led-running = &power_green;
+		led-upgrade = &power_green;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+		pinctrl-names = "default";
+		pinctrl-0 = <&jtag_disable_pins>;
+
+		wlan2g_red {
+			label = "wg800hp:red:wlan2g";
+			gpios = <&gpio 6 GPIO_ACTIVE_LOW>;
+			default-state = "off";
```

Comment:
The "off" state is already the default. You don't need to explicitly set it.

Though, I'm fine with it. Problem is: Somebody is bound to go on a crusade and remove all "default-state = off".

Reply:
Oh, I see. I'll drop it.

Reply:
> Problem is: Somebody is bound to go on a crusade and remove all "default-state = off".

And It just happened. 

https://patchwork.ozlabs.org/patch/1014114/

Good timing.

Reply:
Oh...